### DISCUSSION TITLE
任务迁移

### DISCUSSION THREAD

**Discussion by yangunang** [apache/doris#14019] [A - General / Q&A]
有一个应用场景，我们把1.0集群的数据通过快照的方式恢复到1.1.3集群，现在有没有比较好的方式把1.0的任务一次性迁移到1.1.3上去

**Comment by LemonLiTree** [comment]
具体是什么任务？

**Reply by yangunang** [reply]
导入数据的任务 https://doris.apache.org/zh-CN/docs/data-operate/import/import-way/routine-load-manual?_highlight=%E4%BB%BB%E5%8A%A1#%E5%88%9B%E5%BB%BA%E4%BB%BB%E5%8A%A1

**Comment by dataalive** [comment]
这是两个集群？然后希望将原来指向原来1.0的集群的任务的配置，都能指向新的1.1.3还是？没太理解核心诉求是？

**Reply by yangunang** [reply]
两个集群，希望讲低版本集群的任务（https://doris.apache.org/zh-CN/docs/data-operate/import/import-way/routine-load-manual?_highlight=%E4%BB%BB%E5%8A%A1#%E5%88%9B%E5%BB%BA%E4%BB%BB%E5%8A%A1），导入到高版本集群
